2021-05-03 23:44:15,942 INFO    MainThread:58 [wandb_setup.py:_flush():69] setting env: {}
2021-05-03 23:44:15,943 INFO    MainThread:58 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-03 23:44:15,943 INFO    MainThread:58 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8d3153ad3bb912131270614820cf8ed59c9d94d4'}
2021-05-03 23:44:15,943 INFO    MainThread:58 [wandb_init.py:_log_setup():336] Logging user logs to /content/drive/My Drive/Trash/code/wandb/run-20210503_234415-21cokq5i/logs/debug.log
2021-05-03 23:44:15,944 INFO    MainThread:58 [wandb_init.py:_log_setup():337] Logging internal logs to /content/drive/My Drive/Trash/code/wandb/run-20210503_234415-21cokq5i/logs/debug-internal.log
2021-05-03 23:44:15,945 INFO    MainThread:58 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f074b425810>
2021-05-03 23:44:15,945 INFO    MainThread:58 [wandb_init.py:init():369] calling init triggers
2021-05-03 23:44:15,945 INFO    MainThread:58 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-03 23:44:15,945 INFO    MainThread:58 [wandb_init.py:init():418] starting backend
2021-05-03 23:44:15,945 INFO    MainThread:58 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-03 23:44:15,973 INFO    MainThread:58 [backend.py:ensure_launched():123] starting backend process...
2021-05-03 23:44:16,005 INFO    MainThread:58 [backend.py:ensure_launched():128] started backend process with pid: 333
2021-05-03 23:44:16,009 INFO    MainThread:58 [wandb_init.py:init():423] backend started and connected
2021-05-03 23:44:16,087 INFO    MainThread:58 [wandb_init.py:init():465] updated telemetry
2021-05-03 23:44:16,097 INFO    MainThread:58 [wandb_init.py:init():484] communicating current version
2021-05-03 23:44:16,807 INFO    MainThread:58 [wandb_init.py:init():489] got version response 
2021-05-03 23:44:16,807 INFO    MainThread:58 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-03 23:44:17,039 INFO    MainThread:58 [wandb_init.py:init():522] starting run threads in backend
2021-05-03 23:44:18,531 INFO    MainThread:58 [wandb_run.py:_console_start():1554] atexit reg
2021-05-03 23:44:18,539 INFO    MainThread:58 [wandb_run.py:_redirect():1401] redirect: SettingsConsole.WRAP
2021-05-03 23:44:18,540 INFO    MainThread:58 [wandb_run.py:_redirect():1444] Wrapping output streams.
2021-05-03 23:44:18,540 INFO    MainThread:58 [wandb_run.py:_redirect():1468] Redirects installed.
2021-05-03 23:44:18,540 INFO    MainThread:58 [wandb_init.py:init():546] run started, returning control to user process
2021-05-03 23:44:18,540 INFO    MainThread:58 [wandb_config.py:__setitem__():141] config set learning_rate = 0.01 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f07463f3390>>
2021-05-03 23:44:18,541 INFO    MainThread:58 [wandb_run.py:_config_callback():779] config_cb learning_rate 0.01 None
2021-05-03 23:44:18,541 INFO    MainThread:58 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-03 23:44:18,541 INFO    MainThread:58 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-03 23:44:18,549 INFO    MainThread:58 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-03 23:44:24,126 INFO    MainThread:58 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-03 23:44:24,126 INFO    MainThread:58 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-03 23:44:24,132 INFO    MainThread:58 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-03 23:44:40,476 INFO    MainThread:58 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-03 23:44:40,476 INFO    MainThread:58 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-03 23:44:40,489 INFO    MainThread:58 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-03 23:44:40,517 INFO    MainThread:58 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-03 23:44:40,517 INFO    MainThread:58 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-03 23:44:40,525 INFO    MainThread:58 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-03 23:44:40,555 INFO    MainThread:58 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-03 23:44:40,555 INFO    MainThread:58 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-03 23:44:58,662 INFO    MainThread:58 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-03 23:44:58,668 INFO    MainThread:58 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-03 23:44:58,669 INFO    MainThread:58 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-03 23:45:09,947 INFO    MainThread:58 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-03 23:45:09,974 INFO    MainThread:58 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-03 23:45:09,975 INFO    MainThread:58 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-03 23:45:40,533 INFO    MainThread:58 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-03 23:45:40,709 INFO    MainThread:58 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-03 23:45:40,712 INFO    MainThread:58 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-03 23:45:46,661 INFO    MainThread:58 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-03 23:45:46,681 INFO    MainThread:58 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-03 23:45:46,682 INFO    MainThread:58 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-03 23:45:54,085 INFO    MainThread:58 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-03 23:45:56,988 INFO    MainThread:58 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-03 23:45:56,988 INFO    MainThread:58 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-03 23:45:57,675 INFO    MainThread:58 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-03 23:45:57,690 INFO    MainThread:58 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-03 23:45:57,690 INFO    MainThread:58 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-03 23:46:03,215 INFO    MainThread:58 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-04 04:26:04,731 INFO    MainThread:58 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-04 04:26:04,731 INFO    MainThread:58 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-04 04:33:57,799 INFO    MainThread:58 [wandb_run.py:_atexit_cleanup():1524] got exitcode: 0
2021-05-04 04:33:57,800 INFO    MainThread:58 [wandb_run.py:_restore():1496] restore
2021-05-04 04:33:58,024 INFO    MainThread:58 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 634
  total_bytes: 634
}

2021-05-04 04:33:58,370 INFO    MainThread:58 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 634
  total_bytes: 8207
}

2021-05-04 04:33:58,564 INFO    MainThread:58 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 634
  total_bytes: 8207
}

2021-05-04 04:33:58,666 INFO    MainThread:58 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8207
  total_bytes: 8207
}

2021-05-04 04:33:58,769 INFO    MainThread:58 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8207
  total_bytes: 8207
}

2021-05-04 04:33:58,871 INFO    MainThread:58 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8207
  total_bytes: 8207
}

2021-05-04 04:33:58,972 INFO    MainThread:58 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8207
  total_bytes: 8207
}

2021-05-04 04:33:59,074 INFO    MainThread:58 [wandb_run.py:_wait_for_finish():1646] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8207
  total_bytes: 8207
}

2021-05-04 04:33:59,176 INFO    MainThread:58 [wandb_run.py:_wait_for_finish():1646] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8207
  total_bytes: 8207
}

2021-05-04 04:34:00,458 INFO    MainThread:58 [wandb_run.py:_show_summary():1801] rendering summary
2021-05-04 04:34:00,460 INFO    MainThread:58 [wandb_run.py:_show_history():1839] rendering history
2021-05-04 04:34:00,462 INFO    MainThread:58 [wandb_run.py:_show_files():1868] logging synced files
